packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/callout-links-three-up.twig

```diff
@@ -0,0 +1,38 @@
+{% set headingLevel = calloutLinksThreeUp.headingLevel ?: "3" %}
```

Comment:
should we check for this number? if this is larger than 4 it won't work

Reply:
I just added validation.Thanks!